static/app/types/core.tsx

```diff
@@ -53,7 +53,7 @@ export interface SelectValue<T> extends MenuListItemProps {
    * will be unable to filter to that label. Use this to specify the plain text of
    * the label.
    */
-  plainTextLabel?: string;
+  textValue?: string;
```

Comment:
@evanpurkhiser `plainTextLabel` does the same thing as `react-aria`'s `textValue`, so I'm renaming this to be consistent throughout.